app/config/runtimes/sizes.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+require_once __DIR__ . '/enum.php';
```

Comment:
Why not an import?

Reply:
Updated to use a namespace import

Reply:
Nothing in `/app` is in the autoloader so we need to either add /app to the autoloader for this one file or require the fil